src/main.rs

```diff
@@ -312,6 +313,17 @@ enum UtilCommand {
         #[arg(short, long)]
         fix: bool,
     },
+    /// Clear caches based on conditions.
+    ///
+    /// A cache entry is removed if it matches any of the given conditions.
+    Evict {
+        /// Clear entries with citation keys matching this regex.
+        #[arg(short, long)]
+        regex: Option<String>,
+        /// Clear cached elements predating the provided time.
+        #[arg(short, long)]
+        before: Option<DateTime<Local>>,
```

Comment:
I think we should make these two options mutually exclusive, since I would usually expect two conditions to be intersected, not union-ed.

Reply:
Actually I will just implement the 'and' behaviour instead (I thought about doing this but I was a bit lazy)

Reply:
Excellent!